src/app/components/snippet/snippet.component.ts

```diff
@@ -31,21 +32,13 @@ export class SnippetComponent implements OnInit, OnDestroy {
   @SelectSnapshot(LanguageState)
   public languages: Language[];
 
-  constructor(private fileResource: FileResource) {}
+  constructor(private fileService: FileService) {}
 
   ngOnInit(): void {
-    this.availableLabelsSubscription = this.activeSnippet$.subscribe(snippet => {
-      if (snippet) {
-        this.activeSnippet = snippet;
-
-        this.fileResource
-          .query({ snippet: snippet.pk })
-          .$promise.then(data => {
-            this.files = data;
-          })
-          .catch(error => {
-            console.log(error);
-          });
+    this.availableLabelsSubscription = this.activeSnippet$.subscribe(activeSnippet => {
+      if (activeSnippet) {
+        this.activeSnippet = activeSnippet;
+        this.files$ = this.fileService.fileList(null, null, activeSnippet.pk);
```

Comment:
@sterapps do you know if we can use keyword arguments here?

Reply:
This is a bit strange in the [documentation](https://openapi-generator.tech/docs/generators/typescript-angular/), but it seems that `useSingleRequestParameter=true` will do the job for us

Reply:
Yes useSingleRequestParameter=true does the trick 👍🏼

Reply:
I will create `1.0.2` as a new release where this option is enabled